digits/job.py

```diff
@@ -22,6 +22,7 @@ class Job(StatusCls):
     Base class
     """
     SAVE_FILE = 'status.pickle'
+    NON_PERSISTENT_JOB_DELETE_TIMEOUT_SECONDS = 3600
```

Comment:
@gheinrich I feel like setting job time out would be more class specific then instance specific. Ie all get_weights jobs would have the same timeout, vs individual instances of get_weights jobs.